### PR TITLE
Add support for --sdl command line arg

### PR DESCRIPTION
**Pull request by swoolcock**
- [x] Depends on https://github.com/ppy/osu-framework/pull/3013 and framework update

### CODE REVIEW

**Comment by huoyaoyuan** [comment]
Launch profile for VS and VSC required. And potentially in README.

**Comment by peppy** [comment]
Probably not required as this is a temporary/experimental setting which will be removed once things are ready.

**Comment by bdach** [comment]
I agree, let's not expose this too loudly to avoid getting unnecessary reports about things that still need ironing out.

**Review by peppy — APPROVED** [review]

File: osu.Desktop/Program.cs [hunk 1]
```diff
@@ -22,8 +22,9 @@ public static int Main(string[] args)
         {
             // Back up the cwd before DesktopGameHost changes it
             var cwd = Environment.CurrentDirectory;
+            bool useSdl = args.Contains("--sdl");
```
Comment (UselessToucan) [inline osu.Desktop/Program.cs]: There is no need for the variable since there is only one usage

Comment (swoolcock) [reply osu.Desktop/Program.cs]: It looks cleaner to me.  If you disagree, make a pull request.

**Pull request merged**